### DISCUSSION TITLE
Data Analysis

### DISCUSSION THREAD

**Discussion by Pjatfem** [community/community#61519] [Discussions]
### Select Topic Area

General

### Body

Hello everyone
[Coursera T8Y57K3BUP4G.pdf](https://github.com/community/community/files/12136533/Coursera.T8Y57K3BUP4G.pdf)

I hope this message finds you well. I am writing to express my keen interest in contributing as a volunteer in the field of data analysis and data science.

Recently, I successfully obtained my certificate in data analysis and data science, and I am eager to apply my knowledge and skills in practical, real-world scenarios. As I embark on this new journey, I am enthusiastic about gaining hands-on experience that will not only further develop my expertise but also allow me to make a meaningful impact in the data community.

Coming from a background in architecture, I possess strong analytical abilities, attention to detail, and a passion for problem-solving. I am confident that these transferable skills will be valuable assets in the realm of data analysis.

My objective in seeking a voluntary position is to not only identify the areas where I can excel but also to contribute my time and efforts to a project or organization that aligns with my values and goals.

If your team or organization has any data-related volunteer opportunities available, I would be grateful for the chance to be a part of it. I am ready to commit my time and dedication to contribute effectively and learn from experienced mentors.

Thank you for considering my application. I have attached my resume and certificate for your reference. If any additional information is required, please do not hesitate to contact me.

Looking forward to the prospect of being a part of your esteemed data team and making a positive difference through my voluntary contributions.

Sincerely,

Olufunke Olatunji

**Comment by queenofcorgis** [comment]
Hi @Pjatfem , we're happy you're here 😀

However, we currently do not allow job listings or seeking on the Community on Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

Thanks for your interest 👍
